scripts/exportExcel.gradle

```diff
@@ -174,6 +174,43 @@ use `gradle exportExcel` to re-export files
                     "\"${it.replaceAll('"', '""')}\""
                 }
                 .join(',') + nl, 'UTF-8')
+
+                // fix #1192 https://github.com/docToolchain/docToolchain/issues/1192
+                // remove unnecessary spans which break Asciidoctor rendering
+                def prev = ''
+                def removed = []
+                def useRemoved = true
+                style.eachWithIndex { s, i ->
+                    if (s!="skip") {
+                        if (s.contains('+')) {
+                            def span = s.split('[+]')[0].split('[.]')
+                            if (span[0] != '') {
+                                removed << span[0] + '+' + s.split('[+]')[1]
+                            } else {
+                                removed << s.split('[+]')[1]
+                            }
+                            println i
```

Comment:
the println statement should be removed or contain a userfriendly message if that is necessary

Reply:
good point :-)